execution/v0.1.1/commands.go

```diff
@@ -12,16 +12,14 @@
 //    License for the specific language governing permissions and limitations
 //    under the License.
 
-package execution
+package executionv011
 
-type ExecutionCommand string
+import (
+	common "github.com/cloudbase/garm-provider-common/execution/common"
+)
 
 const (
-	CreateInstanceCommand     ExecutionCommand = "CreateInstance"
-	DeleteInstanceCommand     ExecutionCommand = "DeleteInstance"
-	GetInstanceCommand        ExecutionCommand = "GetInstance"
-	ListInstancesCommand      ExecutionCommand = "ListInstances"
-	StartInstanceCommand      ExecutionCommand = "StartInstance"
-	StopInstanceCommand       ExecutionCommand = "StopInstance"
-	RemoveAllInstancesCommand ExecutionCommand = "RemoveAllInstances"
+	ValidatePoolInfoCommand        common.ExecutionCommand = "ValidatePoolInfo"
```

Comment:
you can add these to the common package as well. Maybe in a separate `const()` block. This way you can avoid 2 imports in the `v0.1.1` package.

Each package does a swirch anyway and only acts on particular commands. There is no overlap in command names, so feel free to move these in the common package as well. 

Add a comment that they are a `v0.1.1` set of commands and it should be fine.

Reply:
Updated it